app/views/mailers/conversation_reply_mailer/conversation_transcript.html.erb

```diff
@@ -1,7 +1,13 @@
 <% @messages.each do |message| %>
   <tr>
     <td>
-      <b><%= message.sender&.try(:available_name) || message.sender&.name || '' %></b>
+      <% sender_name = sender.try(:available_name) || sender&.name || ''%>
+      <% if @inbox.email? %>
+        <b>From:</b> <%= sender_name %> <%= "<#{message.sender&.email}>" %><br>
+        <b>Subject:</b> <%= message.conversation.additional_attributes&.dig('mail_subject') %><br>
+      <% else %>
+        <b><%= sender_name %></b>
+      <% end %>
     </td>
```

Comment:
```
ActionView::Template::Error: undefined local variable or method `sender' for #<ActionView::Base:0x000000000260c0>
/Users/pranavrajs/chatwoot/chatwoot/app/views/mailers/conversation_reply_mailer/conversation_transcript.html.erb:4:in `block in _app_views_mailers_conversation_reply_mailer_conversation_transcript_html_erb___1219565382567652259_77900'
/Users/pranavrajs/chatwoot/chatwoot/app/views/mailers/conversation_reply_mailer/conversation_transcript.html.erb:1:in `each'
/Users/pranavrajs/chatwoot/chatwoot/app/views/mailers/conversation_reply_mailer/conversation_transcript.html.erb:1:in `_app_views_mailers_conversation_reply_mailer_conversation_transcript_html_erb___1219565382567652259_77900'
/Users/pranavrajs/chatwoot/chatwoot/app/mailers/conversation_reply_mailer.rb:48:in `conversation_transcript'
/Users/pranavrajs/chatwoot/chatwoot/app/mailers/application_mailer.rb:81:in `switch_locale'
```

Reply:
sorry about that, I updated the PR and also tested it thoroughly

![image](https://github.com/chatwoot/chatwoot/assets/8219614/dfc6c88c-39a3-4ca6-b0cd-4bf8e1b910a7)